data_transfer/copy_gtex_v8_whole_blood_data.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -ex
+
+gsutil -u tx-adapt -m cp "gs://gtex-resources/GTEx_Analysis_v8_QTLs/GTEx_Analysis_v8_EUR_eQTL_all_associations/Whole_Blood.v8.EUR.allpairs.*" gs://cpg-gtex-main/v8/whole_blood/
+gsutil -u tx-adapt -m cp gs://gtex-resources/GTEx_Analysis_v8_QTLs/GTEx_Analysis_v8_EUR_eQTL_all_associations/Whole_Blood.v8.EUR.allpairs.chr22.parquet gs://cpg-gtex-test/v8/whole_blood/
```

Comment:
It won't make a huge difference, but if you copy from our `main` bucket as the source, we won't have to pay for egress twice.

Reply:
I can do that! Is there a way to ensure the first command finishes before the second is executed?

Reply:
This happens automatically: every line is executed sequentially in a bash script and `gsutil` only returns once the copy is complete. If you're worried about the first call failing, you've also already taken care of that with `set -ex`, which will make the whole script abort and fail as soon as any command returns a status error code.

Reply:
great, that makes sense. Thanks!